apigw-http-api-cors-sam/README.md

```diff
@@ -0,0 +1,60 @@
+# HTTP API with CORS
+
+This pattern contains the source code and supporting files for a serverless application that you can deploy with the SAM CLI. Deploying the SAM file creates an HTTP API with Lambda proxy integration and CORS enabled.
```

Comment:
What are you achieving with this pattern? Can you elaborate more on why all the CORS conf is with allow all? What is the benefit of this configuration?